app-samples/Cognitive-Services-Face-WPF/Sample-WPF/Controls/UIHelper.cs

```diff
@@ -130,13 +130,45 @@ public static IEnumerable<Face> CalculateFaceRectangleForRendering(IList<Detecte
 
             foreach (var face in faces)
             {
+                var Left = (int)((face.FaceRectangle.Left * scale) + uiXOffset);
```

Comment:
Change Left, Top and FaceAngle with first letter of lower case.

Reply:
Done.